packages/visibility_detector/test/widget_test.dart

```diff
@@ -45,7 +44,7 @@ void main() {
     callback: (tester) async {
       final cellKey = demo.cellKey(0, 0);
       final expectedRect =
-          tester.getRect(find.byKey(demo.cellContentKey(0, 0)));
+      tester.getRect(find.byKey(demo.cellContentKey(0, 0)));
```

Comment:
Did you intend to unindent this line?  Please revert this.

Reply:
I am sorry, this must have happened when I commented out that part to only run my own test and later commented it in again. I have used "dart format" (using Android Studio) now to restore the correct formatting.

Reply:
It seems that "dart format" has added four additional blank lines for a few get/set methods. Is that fine or should they be removed again?

Reply:
That's fine.  It does bring to light that the corresponding dartdoc comments are associated with the private fields rather than with the public getters.  However, it's testing code and won't have generated documentation anyway, so it doesn't really matter.